main/src/ca/uwaterloo/flix/language/phase/Parser2.scala

```diff
@@ -2889,6 +2889,36 @@ object Parser2 {
       close(mark, TreeKind.Expr.FixpointWhere)
     }
 
+    private def predicateShape()(implicit s: State): Mark.Closed = {
```

Comment:
maybe predicateWithShape or predicateAndShape?

Reply:
shape to me sounds like the `(_, _, ..)` part, but fair

Reply:
predicateWithArity